modules/social_features/social_follow_user/social_follow_user.install

```diff
@@ -25,3 +27,46 @@ function social_follow_user_install(): void {
   $permissions[] = 'administer social_follow_user settings';
   user_role_grant_permissions('sitemanager', $permissions);
 }
+
+/**
+ * Update profile view modes/block visibility rules, create user follow lists.
+ */
+function social_follow_user_update_11201(): void {
+  // Display hero & statistic block on the following users/followers pages.
+  $config_names = [
+    'block.block.socialblue_profile_hero_block',
+    'block.block.socialblue_profile_statistic_block',
+  ];
+  foreach ($config_names as $config_name) {
+    $config = \Drupal::configFactory()->getEditable($config_name);
+
+    if (!empty($config->getRawData())) {
+      $pages = $config->get('visibility.request_path.pages');
+      $pages .= "\r\n/user/*/following/users";
+      $pages .= "\r\n/user/*/followers";
+      $config->set('visibility.request_path.pages', $pages);
+      $config->save();
+    }
+  }
+
+  // Create follow/following user lists.
+  $configs = [
+    'views.view.followers_user',
+    'views.view.following_users',
+  ];
+  $config_storage = \Drupal::service('config.storage');
+  $config_path = drupal_get_path('module', 'social_follow_user') . '/config/static/';
+  $source = new FileStorage($config_path);
+
+  foreach ($configs as $config) {
+    $config_storage->write($config, (array) $source->read($config));
+  }
+
+  // Update statistic profile view mode.
```

Comment:
@volodymyr-sydor-lemberg We need to add an update of profile view modes to the hook_install as well. Also, I would move it to the separate update hook as well.